data/NikhilRaikwar.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Nikhil Raikwar",
+  "displayStatsPublic": true,
```

Comment:
As of the latest update, we no longer need `displayStatsPublic` property. So please remove this field from your json. 

```suggestion
```